designs/virtualization/4.6/Upload-image-prior-to-creating-VM/Upload Image prior to creating vm.md

```diff
@@ -0,0 +1,37 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Upload Image prior to creating a VM
+
+This PR deals with when a user wants to upload an image prior to creating a virtual machine.
+It needs to be accomplished via the storage/PVC area.
+It should ensure the user understands what the backing PVC does but does not remove the Disk terminology.
+
+The ‘Create PVC’ button will change into a dropdown, so users will be able to select ‘Upload data to Persistent Volume Claim’ from this dropdown. 
+
+![PVC list with drop down button](img/PVC-dropdown.png)
+
+They will get to a new upload screen with an inline info alert that says this PVC will be created using a Data Volume through Container Data importer (CDI).
+User chooses an image to upload and fills in the PVC details: Name, size, storage class, access mode. Some of the fields can be filled out but editing will be optional as well.
+
+![browse to upload](img/Upload-data-to-pvc-1.png)
+
+After the user clicks ‘Upload’ they will get an inline notification with a spinner + text explanation about the PVC creation progress. We’ll use the spinner over the progress bar because we can't really offer progress and there aren't any steps to highlight.
```

Comment:
This should also send them to the details page of the PVC rather than the list ( just determined that at our stakeholders call today :) )
Another question I had was won't there be an in-between state when the PVC has yet to be created? In that case they would be on the page with an inline notification telling them what CDI is doing right?

Reply:
@matthewcarleton about the in-between state: wouldn't it be odd to place that inline notification (when the PVC hasn't created yet) on the PVC details page?

Reply:
I wonder if it should appear on the list view or as we heard in the meeting in Peter's example on a separate page. WDYT?

Reply:
good point - aligning with what Peter has done make sense to me.